extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -103,7 +109,8 @@ export class LinkerManager {
           this.handleAnchorMutation_.bind(this), Priority.ANALYTICS_LINKER);
     }
 
-    return Promise.all(this.allLinkerPromises_);
+    return Promise.all(this.allLinkerPromises_)
+        .then(this.enableFormSupport_.bind(this));
```

Comment:
let's introduce a feature flag for amp-form support here (so we have full control over the launch).

Reply:
Done.